srv/_grains/platform.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from pathlib import Path
+
+FILENAME="/opt/seagate/lr-serial-number"
+
+
+def add_lr_serial_number():
+    """
+    Populates lr-serial-number to grains
+    """
+
+    grains = {"lr-serial-number": None}
+    if Path(FILENAME).is_file():
```

Comment:
Path(FILENAME) could be a variable, this it avoids re-instantiation of same object twice. This adds code efficiency, although the code works either ways